### ISSUE TITLE
Restore top positively affected logits in plots

### ISSUE THREAD

**Issue by DavidUdell** [DavidUdell/sparse_circuit_discovery#81]

**Comment by DavidUdell** [comment]
Do this in a separate branch, after `feature/baulab_params`. Cleaner that way.
